src/ngraph_translateops.h

```diff
@@ -0,0 +1,269 @@
+    const Node* op, VectNg& ng_arg_vec,
+    const std::vector<const Tensor*>& static_input_map,
+    VectNg& subgraph_out_nodes) {
+  const static std::map<
+      const DataType,
+      const std::pair<const std::function<Status(const Node*, ng::element::Type,
+                                                 std::shared_ptr<ng::Node>*)>,
+                      const ngraph::element::Type>>
+      TF_NGRAPH_CONST_MAP = {
+          {DataType::DT_FLOAT, make_pair(MakeConstOp<float>, ng::element::f32)},
+          {DataType::DT_DOUBLE,
+           make_pair(MakeConstOp<double>, ng::element::f64)},
+          {DataType::DT_INT8, make_pair(MakeConstOp<int8>, ng::element::i8)},
+          {DataType::DT_INT16, make_pair(MakeConstOp<int16>, ng::element::i16)},
+          {DataType::DT_INT32, make_pair(MakeConstOp<int32>, ng::element::i32)},
+          {DataType::DT_INT64, make_pair(MakeConstOp<int64>, ng::element::i64)},
+          {DataType::DT_UINT8, make_pair(MakeConstOp<uint8>, ng::element::u8)},
+          {DataType::DT_UINT16,
+           make_pair(MakeConstOp<uint16>, ng::element::u16)},
+          {DataType::DT_BOOL,
+           make_pair(MakeConstOp<bool, char>, ng::element::boolean)}};
+
+  DataType dtype;
+  TF_RETURN_IF_ERROR(GetNodeAttr(op->attrs(), "dtype", &dtype));
+  // For some reason the following do not work (no specialization of
+  // tensorflow::checkpoint::SavedTypeTraits...)
+  // case DataType::DT_UINT32:
+  //   TF_RETURN_IF_ERROR(MakeConstOp<uint32>(op, ng::element::u32,
+  //   &ng_node));
+  //   break;
+  // case DataType::DT_UINT64:
+  //   TF_RETURN_IF_ERROR(MakeConstOp<uint64>(op, ng::element::u64,
+  //   &ng_node));
+  //   break;
+  try {
+    const auto& func_param = TF_NGRAPH_CONST_MAP.at(dtype);
+    TF_RETURN_IF_ERROR(
+        func_param.first(op, func_param.second, &subgraph_out_nodes[0]));
+  } catch (const std::out_of_range&) {
+    return errors::Unimplemented("Unsupported TensorFlow data type: ",
+                                 DataType_Name(dtype));
+  }
+  return Status::OK();
+}
+
+Status TranslateFloorDivOp(const Node* op, VectNg& ng_arg_vec,
+                           const std::vector<const Tensor*>& static_input_map,
+                           VectNg& subgraph_out_nodes) {
+  subgraph_out_nodes[0] =
+      std::make_shared<ng::op::Floor>(ng_arg_vec[0] / ng_arg_vec[1]);
+  return Status::OK();
+}
+
+Status TranslateFloorModOp(const Node* op, VectNg& ng_arg_vec,
+                           const std::vector<const Tensor*>& static_input_map,
+                           VectNg& subgraph_out_nodes) {
+  auto floordiv =
+      std::make_shared<ng::op::Floor>(ng_arg_vec[0] / ng_arg_vec[1]);
+  subgraph_out_nodes[0] = ng_arg_vec[0] - (floordiv * ng_arg_vec[1]);
+
+  // Possible reuse of floordiv:
+  // TranslateFloorDivOp1(op, ng_arg_vec, static_input_map, subgraph_out_nodes);
+  // subgraph_out_nodes[0] = ng_arg_vec[0] - (subgraph_out_nodes[0] *
+  // ng_arg_vec[1]);
+  return Status::OK();
+}
+}
```

Comment:
```
} // namespace ngraph_bridge
} // namespace tensorflow
```

Reply:
I thought clang-format automatically tags that namespace comment.

from what I see in clang-format's source here: 
https://github.com/llvm-mirror/clang/blob/42c483011b56fbd5d5aad7bad53135190b2d693a/lib/Format/Format.cpp#L729
Google style copies LLVM style and modifies some fields. It does not modify the 'FixNamespaceComments' field which is set to true. So clang-format *should* add these comment lines when we apply style/format. FixNamespaceComments's description can be found here: https://clang.llvm.org/docs/ClangFormatStyleOptions.html
It seems with the current combo of clang-format-3.9 + google-style does not do that automatically.

I tried adding "FixNamespaceComments: true" to graph-tf's .clang-format file, but that produced some errors indicating that clang-format-3.9 does not recognize FixNamespaceComments. That seems to indicate that FixNamespaceComments is a new field that is still not supported by clang-format-3.9

TL;DR: clang-format does not add this comment for now. we have to take care to add it on our own

Reply:
Added commit to add comments